package.json

```diff
@@ -38,7 +38,6 @@
     "babel-core": "7.0.0-bridge.0",
     "babel-preset-cozy-app": "1.9.1",
     "bundlesize": "0.18.0",
-    "cozy-ach": "1.30.0",
```

Comment:
For my knowledge, why cozy-ach should be installed globally ? Shouldn't we put a note in the doc about this?

Reply:
Because it is useful across repositories.

> Shouldn't we put a note in the doc about this?

It is mentioned here : https://github.com/cozy/ach#install

Reply:
Sorry I meant a note in Contacts app documentation ?

Reply:
cozy-ach is a tool used globally across cozy, I think we should not put documentation for every tool that we use inside each of our repo, I think the documentation for cozy-ach is easily accessible from https://docs.cozy.io.

Reply:
But you're right, a little mention does not hurt. Done in cd5f6911fc985854bea5982c83b989a1dafbe166.